proto/spire/plugin/server/upstreamauthority/v1/upstreamauthority.proto

```diff
@@ -57,10 +27,36 @@ service UpstreamAuthority {
     // Implementation note:
     // The stream should be kept open in the face of transient errors
     // encountered while tracking changes to the upstream JWT keys as SPIRE
-    // core will not reopen a closed stream until the next JWT key rotation.
+    // server will not reopen a closed stream until the next JWT key rotation.
     rpc PublishJWTKey(PublishJWTKeyRequest) returns (stream PublishJWTKeyResponse);
```

Comment:
This is probably a good opportunity to tweak the names of these RPCs and/or messages. Something that reflects the behavior a little more accurately would be great (not this, but e.g. PublishAndSubscribe..)